app/helpers/components_helper.rb

```diff
@@ -292,4 +292,14 @@ def form_cancel_button
     end
   end
 
+  def chips_skelton
```

Comment:
this need to go the chip component haml file

Reply:
done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/769/commits/608c867bb548ac97e70c2d9313338faaf7188889